config/read.go

```diff
@@ -55,6 +55,11 @@ func getDefaultConfig() *ServerConfig {
 	if apiKey := os.Getenv("PGA_API_KEY"); apiKey != "" {
 		config.APIKey = apiKey
 	}
+	// Since there's a discrepancy here between the config file key (api_base_url) and the
```

Comment:
I wonder a little bit if instead of this comment we should have one before `PGA_API_BASEURL` below that says its deprecated but we keep this for backwards compatibility.

Reply:
Oh, I did not intend to change the "canonical" key here: we'd need to change the docs to do that. But if you think that's worth doing, sure, that makes sense to me (and I'm happy to make the docs change as well).

Reply:
Yeah, I think its worth doing the docs change as well. Maybe with a small note saying "This used to be called PGA_API_BASEURL before collector release 0.XX.X", in case someone is on an old collector version.